virttest/utils_conn.py

```diff
@@ -1777,7 +1784,7 @@ def conn_recover(self):
         client_session = self.client_session
         try:
             libvirtd_service = utils_libvirtd.Libvirtd(session=client_session)
-            if self.libvirt_ver:
+            if self.libvirt_ver and not utils_split_daemons.is_modular_daemon():
```

Comment:
It seems if you missed self.libvirt_ver and utils_split_daemons.is_modular_daemon(), right?

Reply:
@Yingshun I do not know whether I can follow your comments exactly. The following is my answer now.
For the self.libvirt_ver and utils_split_daemons.is_modular_daemon(), it is just the case for the split daemon mode, it will go to the else statement, just do "libvirtd_service.restart()"

Reply:
Okay.  Let me clarify my question. virt*d exposed 3 sockets since libvirt 5.6, so is it necessary to stop virt*d.socket, daemon-reload and start virt*d.socket before restarting virt*d?